docs/source/install/vagrant.rst

```diff
@@ -0,0 +1,121 @@
+.. |br| raw:: html
+
+   <br />
+
+
+Vagrant & Virtual Appliance
+============================
+Vagrant / OVA is a quick and easy way to try |st2| in a VM.
+
+It's already pre-installed, configured and tested virtual appliance image and so saves time avoiding
+time-consuming installation and configuration steps. Perfect for a quick platform overview,
+testing, demo or even using |st2| in isolated from the internet air-gapped systems.
+
+We highly recommend to use a Vagrant box to get familiar with the |st2| platform.
+
+.. contents:: Contents
+   :local:
+
+The source code is available as a GitHub repo:
+`StackStorm/packer-st2 <https://github.com/StackStorm/packer-st2>`_
+
+---------------------------
+
+Requirements
+------------
+`Vagrant <https://www.vagrantup.com/>`_ and `Virtualbox <https://www.virtualbox.org/>`_ are required.
+If you're not familiar with Vagrant, we recommend to take a look at `Introduction to Vagrant <https://www.vagrantup.com/intro/index.html>`_.
+
+Quick Start
+-----------
+Starting a |st2| Vagrant VM is easy:
+
+.. code-block:: bash
+
+  vagrant init stackstorm/st2
+  vagrant up
+  vagrant ssh
+
+
+Virtual Appliance / OVA
+-----------------------
+As alternative to Vagrant box is Virtual appliance which is available for download as ``.OVA``
+image from the `StackStorm/packer-st2 Github Releases <https://github.com/StackStorm/packer-st2/releases>`_
+page. It might be especially helpful for the isolated from the internet air-gapped environments.
+
+.. note::
+
+    *Linux login credentials:* |br|
+    Username: ``vagrant`` |br|
+    Password: ``vagrant`` |br|
+
+    *StackStorm login details:* |br|
+    Username: ``st2admin`` |br|
+    Password: ``Ch@ngeMe`` |br|
```

Comment:
Another idea here is to use `vagrant` as a username and password for everything, incl. stackstorm.

WDYT?

Reply:
There's a bunch of documentation that refers to the st2admin user and its password. Using `vagrant:vagrant` may increase support costs?

Reply:
Agreed. Changing username will be a PITA

Reply:
Makes sense.

I found it's used in a `Quick Start` section: https://docs.stackstorm.com/start.html, - that's important.

The other things are curl|bash + manual install which doesn't apply to Vagrant.